process-engine/management_api_core
# ✨Add getActiveCronjobs UseCase

## Changes

1. Implement `getAllActiveCronjobs` UseCase

## Issues

Part of https://github.com/process-engine/process_engine_runtime/issues/363

PR: #58

## How to test the changes

- Deploy some ProcessModels with cyclic `TimerStartEvents`
- Start the CronjobService
- Access `getAllActiveCronjobs` and see that a list with all cronjobs is returned
